.github/pull_request_template.md

```diff
@@ -0,0 +1,13 @@
+### Please confirm your submission meets all the criteria
```

Comment:
none of the files in .github should be part of your PR - this is why you have to base against the new-pr branch so you don't have to carry this stuff into your final app ;-)